pyanaconda/errors.py

```diff
@@ -106,8 +106,6 @@ def _get_default_mapping(self):
             PayloadInstallError.__name__: self._payload_install_handler,
 
             # DNF errors
-            "NoStreamSpecifiedException": self._no_module_stream_specified,
-            "InstallMoreStreamsException": self._multiple_module_streams_specified,
```

Comment:
It would be great if you link the DNF changes to the commit message but if it would be pain to find those just leave it as it is now. Thanks!